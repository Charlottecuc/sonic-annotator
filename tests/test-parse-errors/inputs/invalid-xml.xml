<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0">
  <channel>
    <title>A Title</title>
    <link>http://example.com</link>
    <description/>
    <!-- NB missing end-tag for channel -->
</rss>
