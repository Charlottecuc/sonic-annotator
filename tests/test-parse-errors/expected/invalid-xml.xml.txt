XML parser reported: tag mismatch
